tor-button@shatur/files/tor-button@shatur/applet.js

```diff
@@ -94,10 +105,10 @@ MyApplet.prototype = {
     
     on_applet_clicked: function() {
         if (torAppletCheck.substr(5,3) == "OFF") { // Check information from file without 'true,' and '\n '
-                                                   GLib.spawn_command_line_async(this.location); // Run tor from location settings        
+                                                   GLib.spawn_command_line_async(gksu+this.location); // Run tor from location settings        
```

Comment:
When I click this, I see:

```
(cinnamon:6258): Cjs-WARNING **: JS ERROR: GLib.SpawnError: Failed to execute child process "tor" (No such file or directory)
MyApplet.prototype.on_applet_clicked@/home/jason/.local/share/cinnamon/applets/tor-button@shatur/applet.js:108:52
Applet.prototype._onButtonPressEvent@/usr/share/cinnamon/js/ui/applet.js:282:21
```
This is because I'm using the latest build from a PPA. The actual command to start Tor for me is ```/usr/bin/tor-browser-en.sh```.